### ISSUE TITLE
Geolocation of hostname

### ISSUE THREAD

**Issue by krmaxwell** [yolothreat/utilitybelt#55]
Rather than resolve a host and then geolocate the IP, we should probably geolocate based on WHOIS.

**Comment by sroberts** [comment]
Or do both? I mean I would say most people who ask a question like that don't really know what they're asking. And the mismatch is always especially interesting.

**Comment by krmaxwell** [comment]
Yeah, we should have both options, agreed. We're providing a library, after all.
